evolver/app/main.py

```diff
@@ -44,31 +44,31 @@ async def describe_evolver():
     }
 
 
-@app.get("/state")
+@app.get("/state", operation_id="state")
 async def get_state():
     return {
         "state": app.state.evolver.state,
         "last_read": app.state.evolver.last_read,
     }
 
 
-@app.post("/")
+@app.post("/", operation_id="update")
 async def update_evolver(config: EvolverConfigWithoutDefaults):
     app.state.evolver = Evolver.create(config)
     app.state.evolver.config_model.save(app_settings.CONFIG_FILE)
 
 
-@app.get("/schema/", response_model=SchemaResponse)
+@app.get("/schema/", response_model=SchemaResponse, operation_id="schema")
 async def get_schema(classinfo: ImportString | None = evolver.util.fully_qualified_name(Evolver)) -> SchemaResponse:
     return SchemaResponse(classinfo=classinfo)
 
 
-@app.get("/history/{name}")
+@app.get("/history/{name}", operation_id="history")
 async def get_history(name: str):
     return app.state.evolver.history.get(name)
 
 
-@app.get("/healthz")
+@app.get("/healthz", operation_id="healthcheck")
```

Comment:
adding `operation_id` means the generated clients have nice names for these operations

### Before
<img width="1481" alt="Screenshot 2024-07-18 at 4 41 16 PM" src="https://github.com/user-attachments/assets/1a698313-fc00-42fd-ae24-29f920b28340">


### After
<img width="1174" alt="Screenshot 2024-07-18 at 4 23 17 PM" src="https://github.com/user-attachments/assets/01a22199-15dc-43a9-a50a-3ebbf164a853">